scripts/ChangesSinceLastRelease.txt

```diff
@@ -188,3 +188,33 @@ Windows.Win32.System.Mmc.View.get_CellContents : CellContents...UInt16** => PBST
 Windows.Win32.System.Mmc.View.get_Memento : Memento...UInt16** => PBSTR
 Windows.Win32.System.Ole.Apis.QueryPathOfRegTypeLib : lpbstrPathName...UInt16** => PBSTR
 Windows.Win32.System.Ole.Apis.VarBstrCat : pbstrResult...UInt16** => PBSTR
+# Untangle window scroll/status enums and flags
+Windows.Win32.Media.DirectShow.IVideoWindow.get_WindowState : WindowState...Int32* => SHOW_WINDOW_CMD*
+Windows.Win32.System.Ole.IPropertyPage.Show : nCmdShow...UInt32 => SHOW_WINDOW_CMD
+Windows.Win32.System.SystemServices.SHOW_WINDOW_STATUS added
+Windows.Win32.System.SystemServices.SHOW_WINDOW_STATUS.SW_OTHERUNZOOM added
+Windows.Win32.System.SystemServices.SHOW_WINDOW_STATUS.SW_OTHERZOOM added
+Windows.Win32.System.SystemServices.SHOW_WINDOW_STATUS.SW_PARENTCLOSING added
+Windows.Win32.System.SystemServices.SHOW_WINDOW_STATUS.SW_PARENTOPENING added
+Windows.Win32.UI.Controls.RichEdit.ITextHost.TxScrollWindowEx : fuScroll...SHOW_WINDOW_CMD => SCROLL_WINDOW_FLAGS
+Windows.Win32.UI.Shell.IShellLinkA.GetShowCmd : piShowCmd...Int32* => SHOW_WINDOW_CMD*
+Windows.Win32.UI.Shell.IShellLinkA.SetShowCmd : iShowCmd...Int32 => SHOW_WINDOW_CMD
+Windows.Win32.UI.Shell.IShellLinkW.GetShowCmd : piShowCmd...Int32* => SHOW_WINDOW_CMD*
+Windows.Win32.UI.Shell.IShellLinkW.SetShowCmd : iShowCmd...Int32 => SHOW_WINDOW_CMD
+Windows.Win32.UI.WindowsAndMessaging.Apis.ScrollWindowEx : flags...SHOW_WINDOW_CMD => SCROLL_WINDOW_FLAGS
+Windows.Win32.UI.WindowsAndMessaging.Apis.SW_MAX added
```

Comment:
Is it expected that this isn't part of any enum?

Reply:
Strange, where that even came from?

Reply:
Ah I see it in winuser.h. Missing from the docs! Will fix.

Reply:
The ConstantScraper picked it up from winuser.h. It used to be autoPopulated into the previous enum, but since it wasn't moved to another, it's now showing up as a loose constant. You can see further below in the diff that SW_MAX was removed from the original enum.